openc3/lib/openc3/conversions/processor_conversion.rb

```diff
@@ -40,6 +40,7 @@ def initialize(processor_name, result_name, converted_type = nil, converted_bit_
       end
       @converted_bit_size = Integer(converted_bit_size) if ConfigParser.handle_nil(converted_bit_size)
       @converted_array_size = Integer(converted_array_size) if ConfigParser.handle_nil(converted_array_size)
+      @params = [@processor_name, @result_name]
```

Comment:
MIssing converted_ params

Reply:
Base class records those ... they are output in the `to_config` method